lib/launch-fastify.js

```diff
@@ -116,7 +116,7 @@ async function launchFastify(file, options, address) {
     .register(fastifySensible, { errorHandler: mergedOptions.errorHandler })
 
   if (options.exposeMetrics) {
-    fastify.register(fp(metricsRoutes), { serviceModule, logLevel: statusLogLevel })
+    fastify.register(fp(metricsRoutes), { serviceModule, mergedOptions, logLevel: statusLogLevel })
```

Comment:
this is passed with key `mergedOptions` can't we call the property simply `options` or spread them in the options object altogether?

Reply:
I've changed. It's better?